src/js/components/search/filters/awardType/AwardType.jsx

```diff
@@ -44,7 +51,8 @@ const propTypes = {
     awardTypes: PropTypes.arrayOf(PropTypes.object),
     awardType: PropTypes.object,
     bulkTypeChange: PropTypes.func,
-    dirtyFilters: PropTypes.symbol
+    dirtyFilters: PropTypes.symbol,
+    restrictChildren: PropTypes.bool
```

Comment:
This property is within the `awardTypes` prop as a property of the objects in the array and can be removed here.